config/locales/simple_form.en.yml

```
@@ -263,7 +263,7 @@ en:
       interactions:
         must_be_follower: Block notifications from non-followers
         must_be_following: Block notifications from people you don't follow
-        must_be_following_dm: Block direct messages from people you don't follow
+        must_be_following_dm: Block direct message notifications from people you don't follow
```

Shouldn't this say:
```suggestion
        must_be_following_dm: Block notifications for private mentions from people you don't follow
```

Given the change to not using "direct messages" in other parts of the UI in favour of "private mentions"?

---

Yes. This sounds good. Thanks!